django-data/image/common/constants.py

```diff
@@ -173,3 +173,34 @@ class SAMPLE_STORAGE_PROCESSING(EnumMixin, Enum):
 DAYS = TIME_UNITS.get_value('days')
 
 KNOWN_STATUSES = ['Error', 'Issues', 'Known', 'Pass', 'Warning']
+
+VALIDATION_MESSAGES_ATTRIBUTES = [
+    ['birth_location', 'birth_location_latitude', 'birth_location_longitude',
+     'birth_location_accuracy'],
+    ['collection_place', 'collection_place_latitude',
+     'collection_place_longitude', 'collection_place_accuracy'],
+    ['animal_age_at_collection_units', 'animal_age_at_collection'],
+    ['preparation_interval_units', 'preparation_interval'],
+    ['availability'],
+    ['storage'],
+    ['storage_processing'],
+    ['term', 'label'],
+    ['organism_part'],
+    ['developmental_stage'],
+    ['physiological_stage'],
+    ['sex'],
+    ['species'],
+]
+
+ONTOLOGY_VALIDATION_MESSAGES = [
```

Comment:
I added also a comment on `submission,views` regarding those message pattern

Reply:
Removed possibility to change ontology terms, just show message to user. This should be changed in another process using Zooma.